src/nvim/CMakeLists.txt

```diff
@@ -118,6 +118,10 @@ endforeach()
 
 list(REMOVE_ITEM NVIM_SOURCES ${to_remove})
 
+if (NOT WIN32)
+  list(REMOVE_ITEM NVIM_HEADERS "${PROJECT_SOURCE_DIR}/src/nvim/os/pty_process_win.h")
+endif()
```

Comment:
Failed "single include test" as you say.
Will you work #7007 after this? Then I close this pull request.